frontend/pages/[resourceType].tsx

```diff
@@ -64,17 +45,26 @@ export default function ResourceList({ ids, resourceType }: InferGetServerSidePr
             marginTop: '10px',
             marginBottom: '20px',
             marginLeft: '150px',
-            marginRight: '150px'
+            marginRight: '150px',
+            height: 1000
           }}
         >
-          {idItems.length > 0 ? ( //if items exist
-            <ul>{idItems}</ul>
+          {resourceInfo.length > 0 ? ( // if items exist
+            <Stack style={{ marginTop: '10px', marginBottom: '10px', marginLeft: '50px', marginRight: '50px' }}>
+              {resourceInfo.map(res => {
+                return (
+                  <div key={res.id}>
+                    <ResourceInfoCard resourceInfo={res} />
+                  </div>
+                );
+              })}
+            </Stack>
           ) : (
-            <text>
+            <Text>
               No <i>{`${resourceType}`}</i> resources available
-            </text>
+            </Text>
           )}
-        </div>
+        </ScrollArea>
```

Comment:
This `ScrollArea` still extends off the page. Annoying component to work with. I've messed with it for a while now and can't find a way to make the resizing all work together smoothly. Would be down to chat about it if you're interested.